src/applications/personalization/profile-2/components/Profile2Router.jsx

```diff
@@ -55,6 +65,30 @@ class Profile2Router extends Component {
     }
   }
 
+  handleDowntimeApproaching = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.downtimeApproaching) {
+      return (
+        <DowntimeApproaching
```

Comment:
Will we get into situations where we show 2 downtime notifications when the user is looking at, for instance,  Personal Information?

Reply:
Nope. The one at this level is a modal that pops up if downtime is scheduled to start in the next hour. The others at the section level appear if downtime is _active_.